### PR TITLE
Step 10. Create and populate DB

### PR DESCRIPTION
**Pull request by akimov228aleksei**
1) Added PostgreSQL database
2) Created a fixture file
3) Updated readme file
4) Updated coverage.yml file
5) Added the coverage settings file

### CODE REVIEW

File: web-app/management/management/settings.py [hunk 1]
```diff
@@ -74,8 +74,12 @@
...
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'DjangoDB   ',
+        'USER': 'postgres',
+        'PASSWORD': '12345',
+        'HOST': 'localhost',
+        'PORT': '5432',
```
Comment (jimmyhollywood) [inline web-app/management/management/settings.py:82]: Actually, such approach of providing credentials is unsecured - you store all these data in repository. You should provide them via environment variables

File: .github/workflows/coverage.yml [hunk 2]
```diff
@@ -34,6 +34,17 @@ jobs:
     strategy:
       matrix:
         python-version: [3.9]
+
+    services:
```
Comment (jimmyhollywood) [inline .github/workflows/coverage.yml:38]: Just wondering, why you add this here?

**Review by jimmyhollywood — APPROVED** [review]

File: .github/workflows/coverage.yml (hunk 2 above)
Comment (akimov228aleksei) [reply .github/workflows/coverage.yml:38]: Without this setting, the yaml file cannot connect to the database

File: web-app/management/management/settings.py (hunk 1 above)
Comment (akimov228aleksei) [reply web-app/management/management/settings.py:84]: Fixed

**Pull request merged**